stubs/pyserial/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,82 @@
+# Error: failed to import
+# =======================
+serial.__main__  # SystemExit
+serial.serialcli  # (IronPython) ModuleNotFoundError: No module named 'System'
+serial.serialjava  # No Java Communications API implementation found
+
+# Error: is not present at runtime
+# ================================
+serial.urlhandler.protocol_spy.FormatLog  # Class exists!
```

Comment:
It looks like these don't actually exist in the latest released version of pyserial, only on their master branch. I'm fine with allowlisting these and waiting for the next release

Reply:
Thanks again. I'll send a fix, the next pyserial release might not come soon.